### ISSUE TITLE
Add footer to all pages by default

### ISSUE THREAD

**Issue by codyduong** [codyduong/codyduong.dev#24]
Instead of whitelisting pages, instead explicitly blacklist them.

See: https://github.com/codyduong/codyduong.dev/blob/5cd93772ff30e757f48dbed96fd967ee7e16c5a4/frontend/web/packages/mono-app/mono-app.tsx#L68-L80
